### ISSUE TITLE
ERROR rendering  PNGs in parralel for dash

### ISSUE THREAD

**Issue by fruchtee** [IzakMarais/reporter#350]
I just made an Test-dashboard, which only contains a Text-Panel. Exporting it to PDF gives me a blank page with the following error:

error rendering PNGs in parralel for dash {Title:test Description: VariableValues: Rows:[] Panels:[{Id:1 Type:text Title:Panel Title GridPos:{H:8 W:12 X:0 Y:0}}]}: error getting panel {Id:1 Type:text Title:Panel Title GridPos:{H:8 W:12 X:0 Y:0}}: Error obtaining render: 500 Internal Server Error

Is this a known issue? Or are there any fixes already?

grafana version: 9.5.3

**Comment by fruchtee** [comment]
Is it possible that the reporter is not compatible with Grafana Version 9.5.x ?

I still couldnt find a fix. Some panels just cant get rendered.

Error obtaining render for panel {Id:18 Type:stat Title: date - time GridPos: {H:2 W:6 X:7 Y:8}}, Status: 500, Retrying after 10s...
Error obtaining render for panel {Id:15 Type:volkov-echarts-panel Title: diameter GridPos: {H:8 W:13 X:0 Y:10}}, Status: 500, Retrying after 10s...
Error obtaining render for panel {Id:9 Type:gauge Title: GridPos: {H:7 W:4 X:0 Y:18}}, Status: 500, Retrying after 10s...
Error obtaining render for panel {Id:11 Type:gauge Title: GridPos: {H:7 W:4 X:4 Y:18}}, Status: 500, Retrying after 10s...
Error obtaining render for panel {Id:6 Type:histogram Title: GridPos: {H:7 W:5 X:8 Y:18}}, Status: 500, Retrying after 10s...

**Comment by markoshorro** [comment]
Hey @fruchtee, I'm observing the same issue with Grafana 9.x. Did you manage to solve this?

EDIT: in my case checking the logs I had a missing dynamic library missing, nothing related with the app itself.

**Comment by fruchtee** [comment]
No I did not find any solution for this. Did you find a fix?

**Comment by markoshorro** [comment]
@fruchtee yeah, in my case it was related with my machine, some missing libraries. You could check your logs (Grafana logs) for more information, it helped me a lot.

**Comment by fruchtee** [comment]
Hi @markoshorro I am actually not finding any missing libraries. Which do you had to add?

**Comment by subhashc** [comment]
here is the solution: https://github.com/grafana/grafana-image-renderer/issues/159

**Comment by tommeeeee** [comment]
HI everyone I've found the solution 

error come from this problem 

1. check  ***Grafana log***
you may seen error from plugin.grafana-image-renderer like this 
```sh
logger=plugin.grafana-image-renderer t=2024-04-09T06:11:56.802277104Z level=error msg="Render request failed" url="http://localhost:3000/d-solo/{you dash board uid}/_?from=now-1h&height=500&panelId=29&theme=light&to=now&width=1000&render=1" error="Error: Failed to launch the browser process!\n/var/lib/grafana/plugins/grafana-image-renderer/chrome/linux-124.0.6351.0/chrome-linux64/chrome: error while loading shared libraries: libgbm.so.1: cannot open shared object file: No such file or directory\n\n\nTROUBLESHOOTING: https://pptr.dev/troubleshooting\n"

```
The log told you that you need mechanism for allocating buffers for graphics rendering. 

2. Install libgbm
Following this command 
```sh
sudo apt-get update
sudo apt-get install -y libgbm-dev
```

Restart grafana service and grafana-report then try again.
